pcrf/include/session.h

```diff
@@ -323,9 +481,17 @@ class GxSession
    RulesList &getRules() { return m_rules; }
    RulesList &getInstalledRules() { return m_installed; }
 
+	RulesReportList &getRulesReport() { return m_rulesreport; }
+
    Subscriber &getSubscriber() { return m_subscriber; }
 
    SMutex &getMutex() { return m_mutex; }
+	
+	GxSessionState* getCurrentState() { return mp_currentstate; }
+   void setCurrentState( GxSessionState* current_state) { mp_currentstate = current_state; }
```

Comment:
deletion of previous state in this setCurrent function 
current state -> new state log